telegram/_chatadministratorrights.py

```diff
@@ -71,15 +70,15 @@ class ChatAdministratorRights(TelegramObject):
         can_pin_messages (:obj:`bool`, optional): :obj:`True`, if the user is allowed to pin
             messages; groups and supergroups only.
         can_post_stories (:obj:`bool`, optional): :obj:`True`, if the administrator can post
-            stories in the channel; channels only.
+            stories to the chat.
```

Comment:
these three are now required. didn't even make it to TGs changelog. see [here](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Bot-API-Backward-Compatibility) part 1 for how that can be handled